python/src/FittingTest_doc.i.in

```diff
@@ -246,7 +246,7 @@ Parameters
 ----------
 sample : 2-d sequence of float
     Tested sample.
-models : list of :class:`~openturns.Distribution` or :class:`~openturns.DistributionFactory`
+models : list of :class:`~openturns.Distribution`
```

Comment:
Why are Distribution Factories no longer allowed?

Reply:
because now you have a FittingTest::BestModelLilliefors() method to do that ;-)! The previous documentation was documenting two different methods with the same name but different arguments, Now you have two different names for these methods.